Storage.java

```diff
@@ -1 +1,33 @@
+package com.company;
 
+import java.util.ArrayList;
+
+public class Storage<T, V> {
+
+    private ArrayList<T> keys = new ArrayList<>();
+    private ArrayList<V> values = new ArrayList<>();
+
+    public void put(T key, V value) {
+        if (!keys.contains(null) && !values.contains(null)) {
```

Comment:
do you need this checking?

Reply:
Oh, i think no, because we can return null. 
I corrected it